dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -21,6 +21,20 @@
     <Content Include="@(Image)" Condition="'$(OutputType)' == 'Exe'" />
   </ItemGroup>
 
+  <!-- The .NET/C# project system does not add the .resw files added to the project to PRIResource, unlike the UWP
+       project system (this is expected since .resw files were historically UWP specific). That is needed for the
+       .resw files to be indexed during PRI generation. Likewise, image files need to be added to Content for them
+       to be indexed. These files exist in the None list by default (meaning without app developer intervention).
+       We address this requirement here so that the app develpoer doesn't need to manually intervene. -->
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

Comment:
>"'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" [](http://example.com/codeflow?start=23&length=49)

not for non-.net app?

Reply:
Yeah, this is specific to .NET.

Reply:
are you going to have different change for non .net, or we are not going to fix non .net, or this problem doesn't apply to non .net?

Reply:
This problem doesn't exist for non .NET as far as we know. For example, with C++/WinRT, this problem doesn't exist.
Note: This problem did exist for classic win32 (OutputType == Exe), but that's addressed already.

Reply:
do you know how it's not a problem for C++? Hopefully .net can be handled the same way.

Reply:
From my past testing for C++/WinRT WinUI:
"
Everything automatically worked:
-	For images, Content was already True. The images showed up in the PRI file.
-	For a resw file, the strings showed up in the PRI file without me modifying the properties.
  -	The default properties:
    -	Content - False
    -	Included in Project - True
"

Reply:
my question was how it automatically works for C++ WinUI app? also have you checked C++ non-WinUI app?

Reply:
It works automatically because these resource files are added to the correct lists because of the default properties.
Yes, I have (a classic win32 app).

Reply:
who sets that default behavior for C++? 
If you look at the C++ templates, there are many types other than classic win32 app.